backend/src/Models/Component.cs

```diff
@@ -0,0 +1,31 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace FormAPI.Models;
+
+public class Component
```

Comment:
can we have a spesefic name instead of `Component`?

Reply:
Agreed, it is a bit vague. I have changed it to FormComponent now. What do you think?

Reply:
It better😊